taxi/tests/test_admin.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminSiteTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="user",
+            password="u1s2e3r4"
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_superuser(
+            username="jo_er",
+            password="z1x2c3v4",
+            license_number="JUY44831"
+        )

```

Comment:
The `self.driver` is created as a superuser, which might not be necessary or reflective of the actual use case if drivers are not intended to be superusers. Consider creating a regular user instead, unless there is a specific reason for making the driver a superuser.